python/tank/deploy/tank_commands/setup_project_params.py

```diff
@@ -1194,7 +1197,32 @@ def get_uri(self):
         :returns: string
         """
         return self._config_uri
-        
+
+    def is_local_configuration(self):
```

Comment:
Previously, "local" meant either a zip file or an existing project, as opposed to a "remote" config in the app store or git. Now, you have split out zip as its own category, so local (as far as I understand) means the config from an existing project. I think we should rename it to `existing_project`, so that we end up with the config types `existing_project/zip/git/app_store`. That feels clearer to me. Do you think that makes sense?